src/services/dashboards/all-dashboards/AllDashboardsPage.vue

```diff
@@ -124,7 +130,12 @@ export default {
                 }
                 return !!(state.dashboardTotalCount && (state.projectDashboardList.length || state.workspaceDashboardList.length));
             }),
-            hasManagePermission: useManagePermissionState(),
+            hasOnlyViewPermission: computed(() => {
+                const projectManagePermission = useManagePermissionState(MENU_ID.DASHBOARDS_PROJECT).value;
+                const workspaceManagePermission = useManagePermissionState(MENU_ID.DASHBOARDS_WORKSPACE).value;
+                return !(projectManagePermission || workspaceManagePermission);
+            }),
+            pagePermission: store.getters['user/pagePermissionMap'],
```

Comment:
question: Does it work as reactive ? 😲
I thought the code below works only.
If it works, it would be far better than using like below 🤓

```typescript
            pagePermission: computed(() => store.getters['user/pagePermissionMap'].value),
```

Reply:
Oh.. I miss it. Thanks!